tests/ci_folder_parsing.py

```diff
@@ -221,6 +255,7 @@ class FileBasedTest(aetest.Testcase):
     """Standard pyats testcase class."""
 
     OPERATING_SYSTEMS = get_operating_systems()
```

Comment:
in this method, I believe you should add 'junos' to the list of operating_systems that get returned if an operating systems isn't specified.